packages/rdma-core/rdma-core.spec

```diff
@@ -0,0 +1,165 @@
+
+BuildRequires: cmake >= 2.8.11
+BuildRequires: %{_cross_os}libnl-devel
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnl
+
+%description
+%{summary}.
+
+%package devel
+Summary: RDMA core development libraries and headers
+# For now we are only providing the libibverbs-devel as it
+# will be needed to build EFA Device Plugin.
+Provides: %{_cross_os}libibverbs-devel
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n rdma-core-%{version} -p1
+
+%build
+%{cross_cmake} . \
+  -DNO_PYVERBS=1 \
+  -DNO_MAN_PAGES=1 \
+  -DENABLE_STATIC=1 \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX:PATH=%{_cross_prefix} \
+  -DCMAKE_INSTALL_LIBDIR:PATH=%{_cross_libdir} \
+  -DCMAKE_INSTALL_BINDIR:PATH=%{_cross_bindir} \
+  -DCMAKE_INSTALL_SBINDIR:PATH=%{_cross_sbindir} \
+  -DCMAKE_INSTALL_INCLUDEDIR:PATH=%{_cross_includedir} \
+  -DCMAKE_INSTALL_SYSCONFDIR:PATH=%{_cross_sysconfdir} \
+  -DCMAKE_INSTALL_FULL_SYSCONFDIR:PATH=%{_cross_sysconfdir} \
+
+%make_build
+
+%install
+%make_install
+
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -p -m 0644 %{S:100} %{buildroot}%{_cross_tmpfilesdir}/rdma-core.conf
+
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
+install -p %{buildroot}%{_cross_sysconfdir}/libibverbs.d/efa.driver %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
+
+%files
+%license COPYING.*
+%{_cross_attribution_file}
+%{_cross_tmpfilesdir}/rdma-core.conf
+%dir %{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
+%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d/efa.driver
+
+# Core EFA libraries
+%{_cross_libdir}/libefa.so.*
+%{_cross_libdir}/libibverbs.so.*
+%{_cross_libdir}/librdmacm.so.*
+%dir %{_cross_libdir}/libibverbs
+%{_cross_libdir}/libibverbs/libefa-rdmav34.so
+
+# Verification tools
+%{_cross_bindir}/ibv_devices
+%{_cross_bindir}/ibv_devinfo
+
+
+# Exclude the bits that are not needed
+%exclude %{_cross_datadir}/perl5
+%exclude %{_cross_docdir}
+%exclude %{_cross_libdir}/udev
+%exclude %{_cross_libexecdir}
+%exclude %{_cross_pkgconfigdir}
+%exclude %{_cross_sbindir}
+%exclude %{_cross_sysconfdir}
+%exclude %{_cross_unitdir}
+
+# Exclude all the unused libs
+%exclude %{_cross_libdir}/*.a
+%exclude %{_cross_libdir}/ibacm*
+%exclude %{_cross_libdir}/libbnxt*
+%exclude %{_cross_libdir}/libcxgb4*
+%exclude %{_cross_libdir}/liberdma*
+%exclude %{_cross_libdir}/libhfi1*
+%exclude %{_cross_libdir}/libhns*
+%exclude %{_cross_libdir}/libibmad*
+%exclude %{_cross_libdir}/libibnetdisc*
+%exclude %{_cross_libdir}/libibumad*
+%exclude %{_cross_libdir}/libmana*
+%exclude %{_cross_libdir}/libmlx*
+%exclude %{_cross_libdir}/libmthca*
+%exclude %{_cross_libdir}/libocrdma*
+%exclude %{_cross_libdir}/libqedr*
+%exclude %{_cross_libdir}/librxe*
+%exclude %{_cross_libdir}/libsiw*
+%exclude %{_cross_libdir}/libvmw*
+%exclude %{_cross_libdir}/rsocket
+
+# Exclude specific RDMA providers (keeping only libefa)
+%exclude %{_cross_libdir}/libibverbs/libbnxt_re-rdmav34.so
```

Comment:
I'd recommend defining a macro for the `rdmav34` part since it may change (per [upstream CMakeLists.txt](https://github.com/linux-rdma/rdma-core/blob/6dd7c8c272cc5dcecec3fb75411fa8328685da59/CMakeLists.txt#L81):
```suggestion
# at top of spec
%global abiver rdmav34

...
%exclude %{_cross_libdir}/libibverbs/libbnxt_re-%{abiver}.so
```